book_source/developers_guide/Adding-an-Input-Converter.Rmd

```diff
@@ -48,14 +49,62 @@ Names should be `met2CF.<sourcename>` and `met2model.<modelname>`.
 * soil moisture for the full column, rather than a layer, is soil_moisture_content
+* A full list of PEcAn standard variable names, units and dimensions can be found here: https://github.com/PecanProject/pecan/blob/develop/base/utils/data/standard_vars.csv
 
-For the standardized files, we are using CF standard names as variable names.
 
 For example, in the [MsTMIP-CRUNCEP](https://www.betydb.org/inputs/280) data, the variable `rain` should be `precipitation_rate`.
 We want to standardize the units as well as part of the `met2CF.<product>` step. I believe we want to use the CF "canonical" units but retain the MsTMIP units any time CF is ambiguous about the units.
 
-The key is to process each type of met data (site, reanalysis, forecast, climate scenario, etc) to the exact same standard. This way every operation after that (extract, gap fill, downscale, convert to a model, etc) will always have the exact same inputs. This will make everything else much simpler to code and allow us to avoid a lot of unnecessary data checking, tests, etc being repeated in every downstream function.
\ No newline at end of file
+The key is to process each type of met data (site, reanalysis, forecast, climate scenario, etc) to the exact same standard. This way every operation after that (extract, gap fill, downscale, convert to a model, etc) will always have the exact same inputs. This will make everything else much simpler to code and allow us to avoid a lot of unnecessary data checking, tests, etc being repeated in every downstream function.
+
+### Adding Single-Site Specific Meteorological Data
+
+Perhaps you have meteorological data specific to one site, with a unique format that you would like to add to PEcAn. Your steps would be to:
+  1. write a script or function to convert your files into the netcdf PEcAn standard
+  2. insert that file as an input record for your site following these [instructions](How to Insert new Input Data)
+  
+### Downloading Met data outside of the workflow
+
+Perhaps you would like to obtain data from one of the sources coupled to PEcAn on its own. To do so you can run PEcAn functions on their own. 
+
+Example 1:
+
+Downloading Amerifluxlbl from Niwot Ridge for the year 2004:
+```
+raw.file <-PEcAn.data.atmosphere::download.AmerifluxLBL(sitename = "US-NR1", 
+                                             outfolder = ".", 
+                                             start_date = "2004-01-01", 
+                                             end_date = "2004-12-31")
+```
+
+If you wanted to convert it to PEcAn Standard you would subsequently execute the following:
+```
+bety = list(user='bety', password='bety',host='localhost', dbname='bety', driver='PostgreSQL',write=TRUE)
+con <- PEcAn.DB::db.open(bety)
+bety$con <- con
+
+in.path <- '.'
+in.prefix <- raw.file$dbfile.name
+outfolder <- '.'
+format.id <- 5000000002
```

Comment:
This bit is magic -- what is this ID and where does it come from?  Maybe break this script up into a couple steps so you can explain more AND provide links to the appropriate documentation (in this case, where in the documentation do we explain how to set up a Format record).  Right now this example doesn't explain how you would use met2CF.csv to process your OWN data.